Libraries/WebSocket/WebSocket.android.js

```diff
@@ -11,28 +11,99 @@
  */
 'use strict';
 
+var RCTDeviceEventEmitter = require('RCTDeviceEventEmitter');
+var RCTWebSocketManager = require('NativeModules').WebSocketManager;
```

Comment:
We don't have a consistent naming convention for modules but the module is not really a "manager". Can you please name it "WebSockets" or similar?